docs/mdbook/src/execution.md

```diff
@@ -0,0 +1,29 @@
+## Execution
+
+Running sugaroid is easy as pie
+
+Just execute
+
+```bash
+>> sugaroid
+```
+
+
+from the Terminal (Linux, Mac OS) and PowerShell (on Windows)
+
+There are few arguments that can be passed to sugaroid
+
+* qt : Running sugaroid qt will start the sugaroid graphical user interface
+* audio : Running sugaroid audio will include audio support for sugaroid (Data charges may apply)
+* train: Running sugaroid train will start the sugaroid trainer, which you can use to train sugaroid for some responses
+* update : Running sugaroid update will clear the current database and train the new data and store it persistently to the configuration path as sugaroid.db . (See Configuration for more details)
```

Comment:
It would be better to surround qt, etc.. with backticks, `qt`, `audio`, `train`, etc..

Reply:
Okay :)